src/typecheck/bitv.mli

```diff
@@ -1,5 +1,25 @@
 
 
+
+(** Alt-Ergo bitvector builtins *)
+module Ae : sig
+
+  module Tff
+      (Type : Tff_intf.S)
+      (Ty : Dolmen.Intf.Ty.Ae_Bitv with type t := Type.Ty.t)
+      (T : Dolmen.Intf.Term.Ae_Bitv with type t := Type.T.t) : sig
+
+    type _ Type.err +=
+      | Invalid_bin_char : char -> Dolmen.Std.Term.t Type.err
+      (** Error raised when a character that isn't '0' or '1' is
+          used inside a bitvector string *)
```

Comment:
You also should add a doc comment for the whole type.

Reply:
Done.